test_global_controller/test_global_controller_verilog_sim.py

```diff
@@ -0,0 +1,46 @@
+import magma as m
+from common.genesis_wrapper import run_genesis
```

Comment:
can you rebase onto master? some of these interfaces have changed

Reply:
done

Reply:
shouldn't this be `from common.run_genesis import run_genesis`? Can you double check that you rebased on master?